CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.7.1 - 2023-03-17
+### Changed
+- Replies from `/findroom` are now sent ephemerally.
+
```

Comment:
I believe there's been quite a few other changes since January, right?

Reply:
Search me, I just approve the PRs lol